Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1450,6 +1451,16 @@ void WebProcess::deleteAllCookies(CompletionHandler<void()>&& completionHandler)
     completionHandler();
 }
 
+void WebProcess::deleteWebsiteDataForOrigin(OptionSet<WebsiteDataType> websiteDataTypes, const ClientOrigin& origin, CompletionHandler<void()>&& completionHandler)
+{
+    if (websiteDataTypes.contains(WebsiteDataType::MemoryCache)) {
```

Comment:
Should we ASSERT that websiteDataTypes.contains(WebsiteDataType::MemoryCache)?
Or are there cases where we send this message for no good reason?

Reply:
I can add an assertion.